### ISSUE TITLE
JS error in IE

### ISSUE THREAD

**Issue by eltatoo** [rickharris/StickyScroll#3]
Hi, I'm getting an js-error in IE. (Even with your demo page)
Line: 72
Char: 13
Error: Object doesn't support this property or method
Code: 0

I get no problems in Firefox en Chrome.

This looks really good ans is exactly what I need unfortunately it also has to work in IE.

**Comment by amurorei83** [comment]
I have made the fix on the "branch". Waiting on owner to merge into "master"
